src/list.tsx

```diff
@@ -652,9 +652,16 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
     );
 });
 
-export const Row = React.forwardRef<TouchableElement, RowContentProps>(({dataAttributes, ...props}, ref) => (
-    <RowContent {...props} ref={ref} dataAttributes={{'component-name': 'Row', ...dataAttributes}} />
-));
+export const Row = React.forwardRef<TouchableElement, RowContentProps>(
+    ({dataAttributes, role = 'listitem', ...props}, ref) => (
```

Comment:
note, it's not needed to do the same for `BoxedRow`, because `BoxedRowList` is using a `Stack` with the list role, which automatically sets the listitem role to their children

Reply:
On of the drawbacks of usign the list role is that is not possible to make a difference between ordered or unordered lists, have you considered using as="ul | ol" in the rowList component instead?

Reply:
hmmm... do you think that is a needed use case? do we need `<ol>` here?
Also, using `<ol>` removing the default styles has it's own problems too:
![image](https://github.com/Telefonica/mistica-web/assets/1849576/df53692a-5c50-45c1-9189-5448723cda67)

Reply:
I was thinking in an invoices list as a use case for `<ol>`, but definitively based on Safari's behaviour the role="list" seems the right choice here yes. Thanks!